modules/recon/credentials-credentials/bozocrack.py

```diff
@@ -38,10 +39,16 @@ def module_run(self, hashes):
             # sleep to avoid lock-out
             time.sleep(random.randint(3,5))
 
+
 def crack(hashstr, wordlist):
     for word in wordlist:
-        for hashtype in hashlib.algorithms:
-            func = getattr(hashlib, hashtype)
-            if func(word).hexdigest().lower() == hashstr.lower():
-                return word, hashtype
+        if word != '':
```

Comment:
Rather than have this check, would it be better to clean up the list before sending it to the function? Was this even necessary? Do blank lines throw errors? If not, then I say leave it as it without the check.

Reply:
Blank lines were throwing errors during the hashing.

Reply:
Then should the blank lines be handled during list processing, or here? I would think during list processing.